el/index.js

```diff
@@ -82,16 +82,66 @@ module.exports = yeoman.Base.extend({
 
     var el;
     var pathToEl;
+    var pathToElements;
+    var pathToBowerComponents;
+    var pathToApp;
+    
+    // Let you specify a path to the app folder
+    if (this.flags.app) {
+
+      // pathToApp = 'subfolder/app'
+      pathToApp = this.flags.app;
+
+      // adding / to the end of the path
+      if (pathToApp.charAt(pathToApp.length - 1) !== '/' ) {
```

Comment:
Have you tested this in Windows? I'm not sure if it'll work as written because they use `\` file paths